### ISSUE TITLE
Create Terraform examples for AWS

### ISSUE THREAD

**Issue by mwallert** [Shift3/standards-and-practices#110]
There should be some examples for setting up different AWS services with Terraform.

- [ ] S3 Bucket
- [ ] SSL Certificate
- [ ] Elastic Beanstalk
- [ ] Cloudfront
- [ ] Route53 Record
- [ ] EC2 Instance

**Comment by coreyshuman** [comment]
@mwallert We need Terraform examples!

**Comment by mwallert** [comment]
@coreyshuman I have some modules I've been working on I'll get something going this week

**Comment by carlosvargas** [comment]
Don't forget that we have an in-progress branch: https://github.com/Shift3/standards-and-practices/tree/ac53b5655b09947150fbc555e4e818d35c1e63a7/aws/terraform/eb-cloudfront

**Comment by mwallert** [comment]
Indeed. I have part of one done for EB that is split into a module but I haven't tested actually running it. I have some time blocked out Friday that I'll start pushing this stuff up.

**Comment by michaelachrisco** [comment]
@mwallert Just wondering, do we have any apps we can use for examples now that we are using Terraform more frequently?

**Comment by mwallert** [comment]
@michaelachrisco The boilerplate and nplh are the two that come to mind.

**Comment by michaelachrisco** [comment]
@mwallert How do you feel about us adding some of the modules from the devops side into the S&P repo?

**Comment by michaelachrisco** [comment]
@mwallert How do you feel about us adding a couple of the devops terraform modules into this repo? That way, the entire org will have those scripts available for them.

**Comment by mwallert** [comment]
@michaelachrisco Yeah absolutely, though that repo made need some permission changes to allow devs to actually see them.
